src/Components/ListCategories.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { getCategories } from '../services/api';
+
+class ListCategories extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      categories: [],
+    };
+  }
+
+  componentDidMount() {
+    this.HandleListCategories();
+  }
+
+  HandleListCategories = () => {
+    getCategories()
+      .then((categories) => this.setState({ categories }));
+  }
```

Comment:
Foi decidido em call no zoom que vamos manter o async/await

```suggestion
  async HandleListCategories = () => {
    const categories = await getCategories();
    this.setState({ categories });
  }
```

Reply:
Obrigado pela sujestão :)